contentgrid-spring-data-rest/src/main/java/com/contentgrid/spring/data/rest/mapping/PlainMapping.java

```diff
@@ -9,13 +9,13 @@
 import org.springframework.beans.factory.annotation.Qualifier;
 
 /**
- * Qualifier annotation for {@link DomainTypeMapping} that relates to search query parameters
+ * Qualifier annotation for {@link DomainTypeMapping} that relates to a plain mapping that does not change property names
  */
 @Target({ElementType.FIELD, ElementType.METHOD, ElementType.PARAMETER, ElementType.TYPE, ElementType.ANNOTATION_TYPE})
 @Retention(RetentionPolicy.RUNTIME)
 @Inherited
 @Documented
 @Qualifier
-public @interface SearchMapping {
-
+public @interface PlainMapping {
```

Comment:
hmm, I was already looking for Spring MVC docs to figure out what the hell `@PlainMapping` was (like a `@GetMapping` variant I did not know about)